src/ripple/app/ledger/MetaView.h

```diff
@@ -65,9 +64,6 @@ enum TransactionEngineParams
 class MetaView : public View
```

Comment:
`MetaView` has a copy constructor that only partially copies the state of `other`. For this reason, we may want to mark the class as `final` and a comment. If someone does need to inherit from `MetaView` they'll be forced to notice the non-standard copy constructor.